site/addons/Imgix/composer.lock

```diff
@@ -69,7 +69,7 @@
             "type": "library",
             "extra": {
                 "branch-alias": {
-                    "dev-master": "1.0.x-dev"
+                    "dev-main": "1.0.x-dev"
```

Comment:
I think this entire file needs to be reverted

Reply:
Good call.